custom/icds_reports/utils/aggregation_helpers/distributed/app_commcare_version.py

```diff
@@ -0,0 +1,118 @@
+from dateutil.relativedelta import relativedelta
+
+from custom.icds_reports.const import AGG_APP_VERSION_TABLE
+from custom.icds_reports.utils.aggregation_helpers import month_formatter, transform_day_to_month, \
+    get_app_version_temp_tablename
+from custom.icds_reports.utils.aggregation_helpers.distributed.base import (
+    BaseICDSAggregationDistributedHelper
+)
+
+
+class AppVersionAggregationDistributedHelper(BaseICDSAggregationDistributedHelper):
```

Comment:
what is this new table used for?

Reply:
This table is used to aggreagate app version and commcare version per awc per month

Reply:
Can these fields not be added to the existing usage queries we have for agg awc, rather than making a new table